pkgs/by-name/me/meteo-qt/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  qt5,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "meteo-qt";
+  version = "3.4";
+
+  src = fetchFromGitHub {
+    owner = "dglent";
+    repo = "meteo-qt";
+    rev = "v${version}";
+    hash = "sha256-J9R6UGfj3vaPfn0vmjeRMsHryc/1pxoKyIE9wteVYbY=";
+  };
+
+  nativeBuildInputs =
+    with python3Packages;
+    [
+      sip
+    ]
+    ++ [
+      qt5.qttools
+      qt5.wrapQtAppsHook
+    ];
+
+  propagatedBuildInputs = with python3Packages; [
+    lxml
+    pyqt5
+  ];
+
+  pythonImportsCheck = [ "meteo_qt" ];
+
+  makeWrapperArgs = [ "\${qtWrapperArgs[@]}" ];
```

Comment:
Also needs `dontWrapQtApps = true`?

Reply:
seems to make no difference? 🤷   
but without `makeWrapperArgs` I get:  
`qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""`

Reply:
> seems to make no difference? 🤷

Find if there's `./result/bin/.meteo-qt_wrapped_wrapped`

Reply:
only `.meteo-qt-wrapped` is there
regardless of `dontWrapQtApps = true;`